### PR TITLE
Set version to `0.1.0+1`

### PR DESCRIPTION
**Pull request by hspinks**
## What does this PR accomplish?

Sets version number to `0.1.0+1`. #301 

## Did you add any dependencies?

N/A

## How did you test the change?

N/A

### CODE REVIEW

**Comment by advayDev1** [comment]
> Do we need to change CFBundleShortVersionString and CFBundleVersion in client/flutter/ios/Flutter/AppFrameworkInfo.plist as well?

Per discussion, it seems not.

File: client/flutter/android/app/src/main/kotlin/org/who/myhealth/MainActivity.kt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.who.myhealth
+package org.who.WHOMyHealth
```
Comment (advayDev1) [inline client/flutter/android/app/src/main/kotlin/org/who/myhealth/MainActivity.kt]: This file needs to chnge directory to WHOMyHealth

Comment (SamMousa) [reply client/flutter/android/app/src/main/kotlin/org/who/myhealth/MainActivity.kt]: It works as is for me.

Comment (SamMousa) [reply client/flutter/android/app/src/main/kotlin/org/who/myhealth/MainActivity.kt]: With the cahgne it also works.

Comment (advayDev1) [reply client/flutter/android/app/src/main/kotlin/org/who/myhealth/MainActivity.kt]: think we should stick with std package naming

**Comment by SamMousa** [comment]
Still works with latest commit on Android.

**Pull request merged**